### PR TITLE
Todolist for Peer review 

### PR DESCRIPTION
**Pull request by souad988**
Create the todolist strucure.
Add Styling to todolist.
Add a new item functionality.
Remove selected item.
Mark a selected item as complete.
Remove all items marked as complete at once.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/58aa8d2b83.js" crossorigin="anonymous"></script>
+    
+    <title>ToDoList</title>
+</head>
+<body>
+    <div class="todolist_container">
+      <div class="todolist_header"><h5>My Todolist</h5>
```
Comment (Ismailco) [inline src/index.html]: - [x] You can use semantic HTML5 instead, for example \<section\> tag

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <i class="fas fa-sync"></i>
+     </div>  
+      <form class="add_task">
+          <input type="text" placeholder=" Add to list..." class="add_task_input"/>
+          <button class="add_task_button"><i class="fas fa-plus"></i></button>
```
Comment (Ismailco) [inline src/index.html]: - [x] You can add a type attribute to buttons. (type="button")

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+if (storageAvailable('localStorage')) {
+  console.log('local storage');
+  if (!localStorage.getItem('todolist')) {
+    populateStorage(todolist.tasks);
+  } else {
+    todolist.tasks = setInputs();
+  }
+}
+
+function controleDeleteIcons(descriptionInputs) {
```
Comment (Ismailco) [inline src/index.js]: - [x] You can use arrow function, for better clean code and ES6

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+      const trush = e.path[1].querySelector('.fa-trash-alt');
+      trush.classList.remove('hide');
+      dots.classList.remove('show');
+      dots.classList.add('hide');
+      trush.classList.add('show');
+    });
+  });
+}
+
+function displayToDoTasks(tasksContainer) {
```
Comment (Ismailco) [inline src/index.js]: - [x] same, you can use arrow function

**Comment by Ismailco** [comment]
Hi @souad988  :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✔️ [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
✔️ [DRY, KISS, and YAGNI](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) rules.
✔️ [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

## Required Changes ♻️

_See the inline comments below for more required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by Ismailco — CHANGES_REQUESTED** [review]
Hi @souad988  :wave:

Good job so far!
There are some issues that you still need to work on but you are almost there!

## To Highlight
✔️ [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
✔️ [DRY, KISS, and YAGNI](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) rules.
✔️ [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

## Required Changes ♻️

_See the inline comments below for more required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by Ismailco — APPROVED** [review]
Hi @souad988,


## Wow, you did it 🎉
Brilliant




### Well done!💪 🥇




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: src/index.js (hunk 4 above)
Comment (souad988) [reply src/index.js]: thanks for mentioning that , i fixed it.

File: src/index.js (hunk 3 above)
Comment (souad988) [reply src/index.js]: thanks your comment is helpful

File: src/index.html (hunk 2 above)
Comment (souad988) [reply src/index.html]: 🙏

File: src/index.html (hunk 1 above)
Comment (souad988) [reply src/index.html]: Hi @Ismailco 
thanks for the time you spent reviewing my work, and for the valuable comments, yes I need to use semantic tags 👍

**Comment by souad988** [comment]
> Hi @souad988,
> 
> ## Wow, you did it 🎉
> Brilliant [ ](https://camo.githubusercontent.com/3561e6e0cd114cab1064a8824bea7636001a7ea92ab4e9ece7a3aeb147a06968/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6433317732347073475965656b435a792f67697068792e676966) Brilliant [ ](https://camo.githubusercontent.com/3561e6e0cd114cab1064a8824bea7636001a7ea92ab4e9ece7a3aeb147a06968/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6433317732347073475965656b435a792f67697068792e676966)
> 
> ### Well done!💪 🥇
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Thanks for the approval @Ismailco

**Pull request merged**